frontend/src/pages/AccountSend.vue

```diff
@@ -590,20 +943,110 @@ function useSendForm() {
       });
       void txNotify(tx.hash, ethersProvider);
       await tx.wait();
-      resetForm();
+      await resetForm();
     } finally {
       isSending.value = false;
       showAdvancedSendWarning.value = false;
     }
   }
 
-  function resetForm() {
+  // Send funds
+  async function onBatchSendFormSubmit() {
+    try {
+      // Form validation
+      for (let i = 0; i < Sends.value.length; i++) {
+        if (!Sends.value[i].receiver || !Sends.value[i].token || !Sends.value[i].amount)
+          throw new Error('please complete the form');
```

Comment:
Also should these errors have i18n translations? Not sure offhand what we normally do there
```suggestion
          throw new Error('Please complete the form');
```

Reply:
Yes! Added translations here https://github.com/ScopeLift/umbra-protocol/pull/491/commits/28cbcfdc97fce7c07c890f793abd1159c6415181